setup.py

```diff
@@ -1,324 +0,0 @@
-# Copyright 2019 The Feast Authors
```

Comment:
maybe we keep this and remove in a separate PR?

Reply:
We have to confirm this will be have with the releases but that should be reasonable to do!

Reply:
Re-added in latest commit